packages/main/src/themes/Bar.css

```diff
@@ -0,0 +1,52 @@
+:host {
+	background-color: var( --sapPageHeader_Background);
+	height: var(--_ui5_bar_base_height);
+	width: 100%;
+}
+
+.ui5-bar {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	height: inherit;
+	width: inherit;
+	background-color: inherit;
+	margin-bottom: 0.5rem;
+	margin-top: 0.5rem;
+}
+
+.ui5-bar .ui5-bar-leftcontent-container {
+	padding-left: 0.5rem;
+}
+
+.ui5-bar .ui5-bar-rightcontent-container {
+	padding-right: 0.5rem;
+}
+.ui5-bar .ui5-bar-midcontent-container {
+	padding-left: 0.5rem;
+	padding-right: 0.5rem;
+}
+
+.ui5-bar.ui5-bar-design-bottomborder{
+	background-color: var(--sapPageHeader_Background);
+	box-shadow: inset 0 -0.0625rem var(--sapPageFooter_BorderColor);
+}
+
+.ui5-bar.ui5-bar-design-topborder{
+	background-color: var(--sapPageFooter_Background);
```

Comment:
At the top you have the following style: 

```css
:host {
	background-color: var( --sapPageHeader_Background);
}
```

and this will let users to change the bar's bg-color and this is great.
But when a design is set, which is always, because we have a default one, you have:

```css
.ui5-bar.ui5-bar-design-topborder {
	background-color: var(--sapPageFooter_Background);
...
}
```

This way you will set a background on the root element inside the Shadow DOM and the users actually won't be able to change the bg-color.

How it should be implemented
```css
:host {
	background-color: var( --sapPageHeader_Background);
}
:host([design="BottomBorder"]) {
	background-color: var(--sapPageHeader_Background); // you can skip that as it is the same bg as  above
}
:host([design="AllBorders"]) {
	background-color: var(--sapPageFooter_Background);
}
:host([design="TopBorder"]) {
	background-color: var(--sapPageFooter_Background);
}

```

Implement all these styles (border, box-shadow, border radius) on the host and just inherit them in the root element:

```css
.ui5-bar-root {
    border-radius: inherit;
	background-color: inherit;
	box-shadow: inherit;
	border: inherit;
}
```